src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitorHelpers.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal class SymbolDisplayVisitorHelpers
+    {
+        internal static IMethodSymbol FindValidCloneMethod(ITypeSymbol containingType)
```

Comment:
>FindValidCloneMethod [](start = 38, length = 20)

Is this method moved from SymbolDisplayVisitor.Types.cs? Please leave it in its original place. Simply make it internal and make other necessary changes, if any.

Reply:
The method was extracted into a separate file, and that file was added as a linked file on the IDE side. We did not want to link SymbolDisplayVistior.Types.cs, since that would involve having to make even more changes to expose additional compiler internals to the IDE.